[INFO   ] Logger: Record log in /storage/emulated/0/Dev/MeusProjetos/Flapy/github/FlapyTeste/.kivy/logs/kivy_19-02-25_22.txt
[INFO   ] Kivy: v1.10.1.dev0, git-Unknown, 20171218
[INFO   ] Python: v2.7.14 (default, Dec 18 2017, 09:30:59) 
[GCC 7.2.0]
[INFO   ] Factory: 194 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_pil, img_gif (img_ffpyplayer ignored)
[INFO   ] KivyMD: KivyMD version: 0.92.0
[INFO   ] Text: Provider: sdl2
[INFO   ] OSC: using <thread> for socket
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL ES 2" graphics system
[INFO   ] GL: Backend used <gl>
[INFO   ] GL: OpenGL version <OpenGL ES 3.2 V@269.0 (GIT@908a5ce, I77d3059488) (Date:06/07/18)>
[INFO   ] GL: OpenGL vendor <Qualcomm>
[INFO   ] GL: OpenGL renderer <Adreno (TM) 506>
[INFO   ] GL: OpenGL parsed version: 3, 2
[INFO   ] GL: Texture max size <16384>
[INFO   ] GL: Texture max units <16>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/storage/emulated/0/Dev/MeusProjetos/Flapy/github/FlapyTeste/main.py", line 185, in <module>
[WARNING] stderr:     
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid2/files/arm-linux-androideabi/lib/python2.7/site-packages/kivy/app.py", line 802, in run
[WARNING] stderr:     self.load_kv(filename=self.kv_file)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid2/files/arm-linux-androideabi/lib/python2.7/site-packages/kivy/app.py", line 599, in load_kv
[WARNING] stderr:     root = Builder.load_file(rfilename)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid2/files/arm-linux-androideabi/lib/python2.7/site-packages/kivy/lang/builder.py", line 301, in load_file
[WARNING] stderr:     return self.load_string(data, **kwargs)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid2/files/arm-linux-androideabi/lib/python2.7/site-packages/kivy/lang/builder.py", line 350, in load_string
[WARNING] stderr:     parser = Parser(content=string, filename=fn)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid2/files/arm-linux-androideabi/lib/python2.7/site-packages/kivy/lang/parser.py", line 393, in __init__
[WARNING] stderr:     self.parse(content)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid2/files/arm-linux-androideabi/lib/python2.7/site-packages/kivy/lang/parser.py", line 502, in parse
[WARNING] stderr:     objects, remaining_lines = self.parse_level(0, lines)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid2/files/arm-linux-androideabi/lib/python2.7/site-packages/kivy/lang/parser.py", line 606, in parse_level
[WARNING] stderr:     level + 1, lines[i:], spaces)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid2/files/arm-linux-androideabi/lib/python2.7/site-packages/kivy/lang/parser.py", line 576, in parse_level
[WARNING] stderr:     'Invalid data after declaration')
[WARNING] stderr: kivy.lang.parser.ParserException: Parser: File "/storage/emulated/0/Dev/MeusProjetos/Flapy/github/FlapyTeste/game.kv", line 6:
[WARNING] stderr: ...
[WARNING] stderr:       4:	BoxLayout:
[WARNING] stderr:       5:		Button:
[WARNING] stderr: >>    6:	birdObj:bird
[WARNING] stderr:       7:	Image:
[WARNING] stderr:       8:		source:'fundo.jpg'
[WARNING] stderr: ...
[WARNING] stderr: Invalid data after declaration
